Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:16.04
```

Comment:
how about 18.04? does 18.04 contains python3 and pip by default?
or python3 image directly?

Reply:
Image size with Ubuntu 16.04 is 629MB, with Ubuntu 18.04 is 667MB. Updated to use Ubuntu 18.04